programs/contractors/contractors.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "name": "Juan Cruz Viotti",
+    "github": "jviotti",
+    "linkedin": "jviotti",
+    "email": "jv@jviotti.com",
+    "website": "https://www.jviotti.com",
+    "type": "tsc",
+    "incorporatedIn": [ "UK", "US" ],
+    "bio": "I'm a TSC member of JSON Schema, did award-winning research at the University of Oxford in the space of binary serialization and compression using JSON Schema, co-authored an O'Reilly book covering JSON Schema, and own several projects in the ecosystem, ranging educational material like LearnJSONSchema.com to a high-performance C++ JSON Schema compiler and validator. I have experience helping organizations be successful with JSON Schema in the Data, API, and IoT spaces all the way from architecture and training to custom development. I'm eager to help you take your JSON Schema integrations to the next level!",
```

Comment:
This is really good.  We should prop each other up by helping write these for each other.

Reply:
Great to hear you liked it and happy to help with other bios if I can!